### ISSUE TITLE
Allow specifying a DNAnexus executable to `dxni`

### ISSUE THREAD

**Issue by jtratner** [dnanexus/dxWDL#338]
Currently dxni finds stubs for _all_ platform applications, which makes it take a while, and is somewhat opaque about how to make wrappers for applets or apps owned by the current user.

It'd be really nice if you could tell dxwdl which applet or application to generate stubs for, e.g., with a `-path` flag as described below.

```
  dxni
    Dx Native call Interface. Create stubs for calling dx
    executables (apps/applets/workflows), and store them as WDL
    tasks in a local file. Allows calling existing platform executables
    without modification. Default is to look for applets.
    options:
      -path                  Generate stub for given DNAnexus path (either app-<app ID> or project:applet-123 or project-123:workflow-123) 
      -apps                  Search only for global apps.
      -o <string>            Destination file for WDL task definitions
      -r | recursive         Recursive search
      -language <string>     Which language to use? (wdl_draft2, wdl_v1.0)
```

Reasons we'd like this:

1. Makes it much easier to write build pipelines for app(let)s that dump WDL files to incorporate them into other pipelines
2. Simplifies testing burden for new applets/applications
3. Clearer/easier interface
4. Enables generating stubs for non-default versions of apps

**Comment by orodeh** [comment]
This sounds reasonable. I am adding it to my TODO list.

**Comment by ying-w** [comment]
Is there any update to this?

**Comment by orodeh** [comment]
I hope to start working on it today.

**Comment by orodeh** [comment]
Merged into master

**Comment by orodeh** [comment]
This was released in version 1.41.
